src/v3/src/types/context.ts

```diff
@@ -42,6 +42,8 @@ export type IWidgetContext = {
   loading: boolean;
   setLoading: StateUpdater<boolean>;
   setWidgetRendered: StateUpdater<boolean>;
+  userIdentifier?: string;
+  setUserIdentifier: StateUpdater<string | undefined>;
```

Comment:
Any thoughts on this name? In v2 it is set as `login_hint`, we don't have to mimic this exactly, but userIdentifier is a bit misleading because it may not actually be the real identifier, and it's only used in specific scenarios. I just don't want this to be confused later down the road with something it isn't.

Reply:
Thats a good point since we are grabbing the username from the input field, it may not necessarily be a valid username.  I believe it was changed because we wanted to make the name more generic.  @shawyu-okta what do you think?

Reply:
Anything wrong with the obvious alternative?

```
  loginHint?: string;
  setLoginHint: StateUpdater<string | undefined>;
```

Reply:
Yeah I had @leemchale-okta change it to `userIdentifier` because I wanted to avoid a very specific global context value if possible, but based on the comments above I don't have an issue reverting to `loginHint`

Reply:
Ahh, that makes sense, my only concern was if a home team eng comes across this, they may think this is some quick way to access the user's identifier. Granted that could be dispelled rather quickly with a codebase search, but just wanted to get your opinion on this. I am good with either way 🙏

Reply:
okay makes sense I will change it to `  loginHint?: string;
  setLoginHint: StateUpdater<string | undefined>;`